core/src/com/goonsquad/galactictd/systems/initialization/PlayScreenInitSystem.java

```diff
@@ -89,44 +96,38 @@ public void fireEvent() {
     }
 
     private void createShips() {
-        for (int i = 0; i < 3; ++i) {
-            final int shipId = archetypeBuilder.buildArchetype(PlayScreenArchetypeBuilder.SHIP_SPRITE);
+        createShip(Color.BLUE, PlayScreen.GAME_WIDTH * 0.50f, PlayScreen.GAME_HEIGHT * 0.80f);
+        createShip(Color.GREEN, PlayScreen.GAME_WIDTH * 0.75f, PlayScreen.GAME_HEIGHT * 0.30f);
+        createShip(Color.RED, PlayScreen.GAME_WIDTH * 0.25f, PlayScreen.GAME_HEIGHT * 0.30f);
+    }
 
-            Renderable shipRenderable = renderableComponentMapper.get(shipId);
-            shipRenderable.texture = gameInstance.assets.manager.get("Owens_Frank.jpg", Texture.class);
+    private void createShip(final Color color, float centerX, float centerY) {
+        final int shipId = archetypeBuilder.buildArchetype(PlayScreenArchetypeBuilder.SHIP_SPRITE);
 
-            Spatial shipPosition = spatialComponentMapper.get(shipId);
-            shipPosition.setBounds(200 * (i + 2), 200 * (i + 2), 200, 200);
+        Renderable shipRenderable = renderableComponentMapper.get(shipId);
+        shipRenderable.texture = shipTexture;
+        shipRenderable.r = color.r;
+        shipRenderable.g = color.g;
+        shipRenderable.b = color.b;
 
-            Rotatable shipRotatable = rotatableComponentMapper.get(shipId);
-            if(i == 1) {
-                shipRotatable.prepareOneTimeRotation(MathUtils.PI * 0.5f);
-            } else {
-                shipRotatable.prepareContinuousRotation();
-            }
+        Spatial shipSpacial = spatialComponentMapper.get(shipId);
+        shipSpacial.spatialType = BoundsType.Circle;
+        shipSpacial.radius = shipRadius;
+        shipSpacial.centerX = centerX;
+        shipSpacial.centerY = centerY;
 
-            RotationSpeed shipRotationSpeed = rotationSpeedComponentMapper.get(shipId);
-            if(i == 1) {
-                shipRotationSpeed.radiansPerSecond = RotationSpeed.VERY_SLOW_ROTATION;
-            } else if(i == 2) {
-                shipRotationSpeed.radiansPerSecond = -RotationSpeed.FAST_ROTATION;
-            } else {
-                shipRotationSpeed.radiansPerSecond = RotationSpeed.FAST_ROTATION;
+        Touchable shipTouchable = touchableComponentMapper.get(shipId);
+        shipTouchable.event = new Event() {
+            @Override
+            public void fireEvent() {
+                // Add this entity to the user-initiated movement system
+                Gdx.app.log("Ship " + color.toString(), "Touched");
```

Comment:
Oh right, that's how that works, okay then. I would have preferred for it to say like, Red Ship or Ship Red or something instead of the hex value, but it isn't of any real importance.